src/test/java/uk/gov/hmcts/reform/hmc/controllers/HearingManagementControllerTest.java

```diff
@@ -272,26 +314,42 @@ private HearingResponse generateHearingResponse() {
 
     /**
      * generate Create Hearing Request.
+     *
      * @return hearingRequest Create Hearing Request
```

Comment:
Here and line 301 - these Javadocs add no value, plus the one on line 316 make no sense - I would personally remove both Javadocs

Reply:
removed

Reply:
updated in 0ae612a810d04011ea9c0e28759e449f3a4fa43d